From b6264d70b0be38ac140fdeb82ac0d22c5f4f7f35 Mon Sep 17 00:00:00 2001
From: Evgeniy A. Dushistov <dushistov@mail.ru>
Date: Sat, 15 Aug 2020 15:09:08 +0300
Subject: [PATCH] fix build on Arch Linux

qtimageformats/src/plugins/imageformats/jp2/qjp2handler.cpp:844:41:
error: no declaration of «pow»
  844 |     const double jasperRate = minRate + pow((double(quality) / double(maxQuality)), 2) * maxRate;

Pick-to: 5.15
Change-Id: I085996c2db2251903b2a3e52e6e648831637c8f9
---

diff --git a/src/plugins/imageformats/jp2/qjp2handler.cpp b/src/plugins/imageformats/jp2/qjp2handler.cpp
index e9a470c..5082023 100644
--- a/src/plugins/imageformats/jp2/qjp2handler.cpp
+++ b/src/plugins/imageformats/jp2/qjp2handler.cpp
@@ -45,6 +45,7 @@
 #include "qcolor.h"
 
 #include <jasper/jasper.h>
+#include <math.h> // for pow
 
 QT_BEGIN_NAMESPACE
 
